roles/StackStorm.redis/meta/main.yml

```diff
@@ -0,0 +1,27 @@
+---
+galaxy_info:
+  description: Install redis as Stackstorm coordination backend
+  author: armab
+  company: StackStorm
+  license: Apache 2.0
+  min_ansible_version: 2.5
+  platforms:
+    - name: Ubuntu
+      versions:
+        - bionic
+        - focal
+    - name: EL
+      versions:
+        - 7
+        - 8
+  galaxy_tags:
+    - system
+    - stackstorm
+    - st2
+    - automation
+    - remediation
+    - devops
+dependencies:
+  - role: StackStorm.epel
+    when: ansible_facts.os_family == 'RedHat'
+  - role: StackStorm.st2repo
```

Comment:
Why does the `StackStorm.redis` role need the st2repo?

Reply:
You are right, there is no technical dependency between both. Removed it from the meta file now.